oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCheckCommand.java

```diff
@@ -486,22 +502,31 @@ private void binaryProperties(NodeState state, String path, BufferedWriter write
             for (PropertyState p : state.getProperties()) {
                 String propPath = PathUtils.concat(path, p.getName());
                 try {
+                    String id ;
                     if (p.getType() == Type.BINARY) {
-                        count.incrementAndGet();
+                        id = p.getValue(Type.BINARY).getContentIdentity();
+                        // Ignore inline encoded binaries in document mk and null references in segment mk
+                        if (id == null || id.startsWith(INLINE_BINARY_SUFFIX)) continue;
```

Comment:
I think this is a Blob... would it be possible to use isInlined() instead of startsWith? That way, INLINE_BINARY_SUFFIX is not needed.